test/TestUtilities/Test.Utility/Signing/SigningTestUtility.cs

```diff
@@ -543,6 +544,64 @@ public static TrustedTestCert<TestCertificate> GenerateTrustedTestCertificateTha
             return TestCertificate.Generate(actionGenerator).WithTrust(StoreName.Root, StoreLocation.LocalMachine);
         }
 
+        public static bool AreVerifierSettingsEqual(SignedPackageVerifierSettings first, SignedPackageVerifierSettings second)
```

Comment:
Why not just have `SignedPackageVerifierSettings` implement `IEquatable<SignedPackageVerifierSettings>`?

Reply:
Because currently, this comparison is only needed for tests.